.rubocop.yml

```diff
@@ -1,7 +1,18 @@
 AllCops:
+  Include:
+    - Capfile
+    - Gemfile
+    - Rakefile
+    - config.ru
+    - '**/*.rb'
   Exclude:
     - 'config/**/*'
-    - 'db/**/*'
+    - 'db/schema.rb'
```

Comment:
OK, but you want to include migrations?

Reply:
Sure, why not? They're ruby files authored by developers. It's just that we have no control over the content of the schema file.

Reply:
I guess it's just that sometimes they're auto-generated, but then again.. we _can_ edit them so why not.
k